### DISCUSSION TITLE
URL Asset wont load on Hubs Cloud

### DISCUSSION THREAD

**Discussion by tttigo** [mozilla/hubs#3652] [Hubs Cloud Questions] [answered]
Hello

I'm having some troubles setting urls to work on my hubs cloud. I created the map on spoke at hubs.mozilla.com, and today I imported it to my hubs cloud.

The same link works fine on hubs.mozilla.com, but on my cloud it wont.

```
Error adding media Error: Error resolving url "https://merch.coquetelmolotov.com.br/": Not Found
    t media-utils.js:54
    v hub-42507308963211fe2909.js:1
    r hub-42507308963211fe2909.js:1
media-loader.js:627:6
    update media-loader.js:627
    v hub-42507308963211fe2909.js:1
    o hub-42507308963211fe2909.js:1
```
![image](https://user-images.githubusercontent.com/6611885/103711942-4c148000-4f97-11eb-8be6-d42a1d24ef42.png)

![image](https://user-images.githubusercontent.com/6611885/103711956-533b8e00-4f97-11eb-81b7-3bb49f962253.png)

Also, Twitter and Instagram URLs wont work..

Thanks

**Comment by robinkwilson** [comment, marked as answer]
Thanks for the screenshots for both hubs.mozilla.com and HC! 

Good news! Since it's working on hubs.mozilla.com, that means on the next HC update this month, it will work on your HC instance. You'll just need to wait for the automatic update for Hubs Cloud (AND if you have a custom client, merge the latest hubs-cloud branch from the hubs repo into your custom client).

We're aware of Twitter urls not working. You can follow that issue here: https://github.com/mozilla/hubs/issues/3654

**Comment by tttigo** [comment]
Thanks for answering.

I really need to get this working for the next week. There's no alternative way to make this work?

**Reply by robinkwilson** [reply]
Unfortunately not really. You could try screen-sharing the website or using a screenshot then putting the link behind the image.
